src/main/java/com/azure/cosmos/examples/autoscalecontainercrud/sync/AutoscaleContainerCRUDQuickstart.java

```diff
@@ -0,0 +1,195 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.examples.autoscalecontainercrud.sync;
+
+import com.azure.cosmos.ConsistencyLevel;
+import com.azure.cosmos.CosmosClient;
+import com.azure.cosmos.CosmosClientBuilder;
+import com.azure.cosmos.CosmosContainer;
+import com.azure.cosmos.CosmosDatabase;
+import com.azure.cosmos.CosmosPagedIterable;
+import com.azure.cosmos.examples.changefeed.SampleChangeFeedProcessor;
+import com.azure.cosmos.examples.common.AccountSettings;
+import com.azure.cosmos.models.CosmosContainerProperties;
+import com.azure.cosmos.models.CosmosContainerRequestOptions;
+import com.azure.cosmos.models.CosmosContainerResponse;
+import com.azure.cosmos.models.CosmosDatabaseRequestOptions;
+import com.azure.cosmos.models.CosmosDatabaseResponse;
+import com.azure.cosmos.models.FeedOptions;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class AutoscaleContainerCRUDQuickstart {
+
+    private CosmosClient client;
+
+    private final String databaseName = "AzureSampleFamilyDB";
+    private final String containerName = "FamilyContainer";
+
+    private CosmosDatabase database;
+    private CosmosContainer container;
+
+    protected static Logger logger = LoggerFactory.getLogger(SampleChangeFeedProcessor.class.getSimpleName());
+
+    public void close() {
+        client.close();
+    }
+
+    /**
+     * Sample to demonstrate the following AUTOSCALE container CRUD operations:
+     * -Create
+     * -Update throughput
+     * -Read by ID
+     * -Read all
+     * -Delete
+     */
+    public static void main(String[] args) {
+        AutoscaleContainerCRUDQuickstart p = new AutoscaleContainerCRUDQuickstart();
+
+        try {
+            logger.info("Starting SYNC main");
+            p.autoscaleContainerCRUDDemo();
+            logger.info("Demo complete, please hold while resources are released");
+        } catch (Exception e) {
+            e.printStackTrace();
+            logger.error(String.format("Cosmos getStarted failed with %s", e));
+        } finally {
+            logger.info("Closing the client");
+            p.shutdown();
+        }
+    }
+
+    private void autoscaleContainerCRUDDemo() throws Exception {
+
+        logger.info("Using Azure Cosmos DB endpoint: " + AccountSettings.HOST);
```

Comment:
http://www.slf4j.org/faq.html#paramException
use slfrj place holder rather than string concat. here and other places.
http://www.slf4j.org/faq.html#paramException